File: app/src/main/java/com/kode/utair/data/network/responses/CitiesListResponse.java
package com.kode.utair.data.network.responses;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class CitiesListResponse {

    @SerializedName("results")
    private List<City> cities;

    public List<City> getCities() {
        return cities;
    }

    public class City {

        @SerializedName("city")
        private String name;

        public String getName() {
            return name;
        }
    }

}

File: app/src/main/java/com/utair/data/network/responses/CitiesListResponse.kt
package com.utair.data.network.responses

import com.google.gson.annotations.SerializedName

class CitiesListResponse {

    @SerializedName("results")
    lateinit var cities: List<City>
        private set

    inner class City {
        @SerializedName("city")
        lateinit var name: String
            private set
    }

}